great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -74,27 +79,43 @@ def __init__(
             project_config (DataContextConfig): config for CloudDataContext
             runtime_environment (dict):  a dictionary of config variables that override both those set in
                 config_variables.yml and the environment
-            ge_cloud_config (GeCloudConfig): GeCloudConfig corresponding to current CloudDataContext
+            cloud_config (GXCloudConfig): GXCloudConfig corresponding to current CloudDataContext
         """
-        self._ge_cloud_mode = True  # property needed for backward compatibility
-
-        self._ge_cloud_config = self.get_ge_cloud_config(
-            ge_cloud_base_url=ge_cloud_base_url,
-            ge_cloud_access_token=ge_cloud_access_token,
-            ge_cloud_organization_id=ge_cloud_organization_id,
+        self._cloud_mode = True  # property needed for backward compatibility
+
+        # If any deprecated args deviate from their defaults, we should flag the issue
+        if (
+            ge_cloud_base_url is not None
+            or ge_cloud_access_token is not None
+            or ge_cloud_organization_id is not None
```

Comment:
Again, wouldn't this be better to use `ge_cloud_base_url or ge_cloud_access_token or ge_cloud_organization_id` an empty string is passed?

Reply:
yeah that's fair! I can change that